src/components/AddButton.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function AddButton() {
+  return (
+    <button type="submit" className="btn">Add Book</button>
+  );
+}
+
+export default AddButton;
```

Comment:
[OPTIONAL] it is a good practice to create reusable components but only when it implements a new functionality or it wraps many lines of code that you will reuse but creating a new component just to call a button is not necessary